mysql-test/suite/innodb/t/alter_charset.test

```diff
@@ -0,0 +1,48 @@
+--source include/have_innodb.inc
+--source include/innodb_row_format.inc
```

Comment:
While many of the test cases in this file do not support `ALGORITHM=INSTANT`, I think that it would be better to move these tests to `innodb.instant_alter_charset`.